metadata-jobs/mae-consumer-job/src/main/resources/application.properties

```diff
@@ -1,3 +1,6 @@
 server.port=9091
 management.endpoints.web.exposure.include=metrics, health, info
 spring.mvc.servlet.path=/
+management.health.elasticsearch.enabled=false
```

Comment:
Have to add since in the latest version, spring checks dependent components, and neo4j status returns DOWN when not being used.

Reply:
this is interesting